server/routes/dashboardPage.tsx

```diff
@@ -15,7 +15,10 @@ app.get(['/dash/pages/:subMode'], async (req, res, next) => {
 		}
 		const initialData = await getInitialData(req, { isDashboard: true });
 
-		if (!initialData.scopeData.activePermissions.canView) {
+		if (
+			!initialData.scopeData.activePermissions.canView ||
+			!initialData.scopeData.elements.activeTarget
+		) {
```

Comment:
I think this should be `NotFoundError` like the others to avoid leaking whether a page exists to a logged out (or otherwise unauthorized) user.

Also I'm still seeing a crash at [/dash/pages/some-bad-name](https://pubpub-pipel-fix-collec-sudbuf.herokuapp.com/dash/pages/some-bad-name) but only while logged in.

Reply:
Agree, that should be separated!

Also that was not properly catching the error (a page is never a target/scope, the activeTarget here is the community so it would pass), but should be fixed now!